website/src/blogs/getting-started-with-k3s-in-vsphere-and-openebs-cstor.md

```diff
@@ -0,0 +1,280 @@
+---
+title: Getting started with K3s in vSphere and OpenEBS cStor
+author: Giridhara Prasad
+author_info: Lead Engineer at Mayadata Inc. Giridhar is experienced in software test automation, chaos engineering. Currently, he's working on Litmus, an Open Source chaos engineering project.
+excerpt: In this blog, more of a tutorial, I will walk you through the steps to install K3OS and setup OpenEBS.
+tags: OpenEBS
+date: 26-03-2020
+---
+
+[K3OS](https://github.com/rancher/k3os/) is a Linux distribution built to run lightweight Kubernetes clusters called [K3s](https://github.com/rancher/k3s/). It is specifically designed only to have what is needed to run [k3s](https://github.com/rancher/k3s).
+
+In this blog, more of a tutorial, I will walk you through the steps to install K3OS and setup OpenEBS, a CNCF project, and leading Open Source Container Attached Storage solution for Kubernetes Stateful Workloads.
+
+#### **Setting up K3OS in vSphere**
+
+K3OS kernel is forked from Ubuntu-18.04 LTS, and its userspace binaries are from alpine. So, you need to select Ubuntu Linux (64 bit) as the guest operating system while creating a virtual machine.
+
+![Select guest operating system](/images/blog/2020/03/114---getting-started-with-k3s-in-vsphere-and-use-openebs-cstor-for-its-persistent-storage.png)
+
+Download the latest K3OS iso file (currently v0.9.0) from its [GitHub release](https://github.com/rancher/k3os/releases) page. Attach the iso file into a virtual machine and start it with the live installation option, as shown below.
+
+Select the option *K3OS LiveCD & install* and boot the operating system.
+
+![Live installation](/images/blog/2020/03/114---getting-started-with-k3s-in-vsphere-and-use-openebs-cstor-for-its-persistent-storage-3.png)
+
+After booting up successfully, you will be landed in a login prompt. The default user in K3OS is rancher. You can login as rancher user without any password.
+
+![Login prompt](/images/blog/2020/03/114---getting-started-with-k3s-in-vsphere-and-use-openebs-cstor-for-its-persistent-storage-6.png)
+
+After performing a live install, You need to install the Operating system into a disk and can configure the machine either as a server(Master) or an agent(worker). This can be performed by executing the command sudo k3os install.
```

Comment:
```suggestion
After performing a live install, You need to install the Operating System into a disk and can configure the machine either as a server(Master) or an agent(worker). This can be performed by executing the command `sudo k3os install`.
```